rest/admin_api_auth_test.go

```diff
@@ -478,518 +478,420 @@ func TestAdminAPIAuth(t *testing.T) {
 		t.Skip("This test only works against Couchbase Server")
 	}
 
+	rt := NewRestTester(t, &RestTesterConfig{
+		adminInterfaceAuthentication:   true,
+		metricsInterfaceAuthentication: true,
+	})
+	defer rt.Close()
+
+	// init RT bucket so we can get the config
+	_ = rt.Bucket()
+	dbConfig := rt.DatabaseConfig
```

Comment:
This test is making authorized as well as unauthorized requests for most endpoints (determined by whether `SkipSuccessTest` is set)

The test was `PUT`ing blank database config, which effectively moves the database's bucket from `sg_test_...` to a `db` bucket (bucket name is inherited if unspecified), which ultimately fails and returns a 502 to pass the `!= 401/403` assertion but then results in a missing database.

This then causes 404s on all other sub-tests that come after, rather than the expected 401s.
The change here is to basically PUT the existing database config (including the correct bucket name).